phalcon/Acl/Adapter/Memory.zep

```diff
@@ -336,112 +437,13 @@ class Memory extends Adapter
         let this->roles[] = roleObject;
         let this->rolesNames[roleName] = true;
 
-        if accessInherits != null {
+        if (null !== accessInherits) {
```

Comment:
Why do we need parentheses here and below?

Reply:
No reason - forgot that we don't need these so force of habit I guess :)